### ISSUE TITLE
override main_files

### ISSUE THREAD

**Issue by rstormsf** [rharriso/bower-rails#160]
main_files only adds some files, would be great to override it

**Comment by SergeyKishenin** [comment]
Have you seen:

```ruby
# via argument
asset "moment", "2.10.1", main_files: ["./locale/en-gb.js"]

# or in block
asset "moment", "2.10.1" do
  main_files [
    "./locale/en-gb.js",
    "./locale/fr.js",
    "./locale/lv.js"
  ]
end
```

at [Ruby DSL Configuration](https://github.com/rharriso/bower-rails#ruby-dsl-configuration)

**Comment by rstormsf** [comment]
Yes, I have seen this and it doesn't seem like you read what I said.
Main_files only adds files to current main object in bower.json
For example, if I had something in my main object by default and then I provide via Bowerfile, it doesn't override it, it will keep whatever is already declared in main object and add what I provided.
This is inconvenient behaviour.

**Comment by SergeyKishenin** [comment]
Well, your issue description is quiet poor, so maybe I missed something. So you want to have an option that will either merge main files declared by you, or override, like

```ruby
asset "moment", "2.10.1" do
  main_files [ "./locale/lv.js" ], override: true # will keep only ./locale/lv.js file
end
```
?

**Comment by rstormsf** [comment]
I'm sorry if I didn't make it clear at first. Yes, that `override: true` flag would be great to have.
You got the idea right now.

**Comment by rstormsf** [comment]
One of the reasons why - is because 
1) when I want to completely ignore package to be installed in `bower_components` folder, but it will be done because it's dependency of bower package. Things like jquery that I load in rails assets pipeline.
2)Maintainers are frequently include more stuff in main object than needed.

**Comment by SergeyKishenin** [comment]
I'll reopen for now until I have more free time or someone provide a PR.
